### PR TITLE
Create a markdown gist for better emoji rendering

### PR DESCRIPTION
**Pull request by muan**
Before

https://gist.github.com/anonymous/afb5521c27f5104e21415ea7dd846a68

mixed not-colored emoji

After

https://gist.github.com/anonymous/625da4e32b61f0c7f9aeddb02c58c371

all colored emoji

:v:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -122,8 +122,8 @@ <h1 title="emoji + pixellate = emojillate">~~emojillate~~</h1>
     clearStatus();
     var result = document.querySelector('.statusText').textContent = 'Creating gist...';
 
-    var fileContents = document.querySelector('.output').innerText;
-    var fileName = document.querySelector('pixel-er').tldr;
+    var fileContents = document.querySelector('.output').innerText.replace(/\n/g, '    \n');
```
Comment (muan) [inline index.html:125]: In case you didn't know 4 spaces create a magical line break in markdown.

**Comment by notwaldorf** [comment]
omg 😍😍😍

File: index.html (hunk 1 above)
Comment (notwaldorf) [inline index.html:125]: of course they would

**Comment by notwaldorf** [comment]
Ughhhh this only works for small-ish images. If they're too [wide](https://gist.github.com/anonymous/bdb1d6edc7708d1ed345f9215ec0c57f) the this doesn't work :(

**Pull request merged**